miden-crypto/src/hash/mod.rs

```diff
@@ -26,15 +26,16 @@ pub mod rpx {
     pub use super::algebraic_sponge::rescue::Rpx256;
 }
 
-mod algebraic_sponge;
+pub mod algebraic_sponge;
```

Comment:
This may have already been covered in some of the discussions, but why do we need to make this module `pub`? Originally, it was meant to be just for internal organization purposes.

Reply:
See https://github.com/0xMiden/crypto/pull/720#discussion_r2649581757